corehq/apps/users/util.py

```diff
@@ -73,6 +73,7 @@ def raw_username(username):
     Strips the @domain.commcarehq.org from the username if it's there
     """
     sitewide_domain = settings.HQ_ACCOUNT_ROOT
+    username = str(username)
```

Comment:
do you need to check for blank values?
https://github.com/dimagi/commcare-hq/pull/25972/files#diff-cceb43df05c432667c703e21a410f57dR149

```
In [1]: str(None)
Out[1]: 'None'
```

Reply:
It already would have errored out on the `lower`, but good point. 08fb05f423b

Reply:
ya thats the reason we added it in the other PR